docs/user-quick-start.md

```diff
@@ -0,0 +1,299 @@
+   Tested with version v1.27.4.
+
+1. Install `clusteradm` tool. See
+   [Install clusteradm CLI tool](https://open-cluster-management.io/getting-started/installation/start-the-control-plane/#install-clusteradm-cli-tool) for the details.
+   Tested with v0.6.0.
+
+1. Install `subctl` tool, See
+   [Submariner subctl installation](https://submariner.io/operations/deployment/subctl/) for the details.
+   Tested with version v0.15.2.
+
+1. Install the `velero` tool. See
+   [Velero Basic Install](https://velero.io/docs/v1.11/basic-install/) for the details.
+   Tested with version v1.11.0.
+
+1. Install `helm` tool - on Fedora you can use:
+
+   ```
+   sudo dnf install helm
+   ```
+   
+   Tested with version v3.11.
+
+1. Install the `virtctl` tool. See
+   [virtctl install](https://kubevirt.io/quickstart_minikube/#virtctl)
+   for the details.
+   Tested with version v1.0.1.
+
+1. Install `docker` - on Fedora you can use:
+
+   ```
+   sudo dnf install docker
+   ```
+
+   Add yourself to the `docker` group to allow running docker as root:
+
+   ```
+   sudo usermod -aG docker $USER && newgrp docker
+   ```
+
+   Restart docker service to fix the permissions on the docker daemon
+   socket:
+
+   ```
+   sudo systemctl restart docker
+   ```
+
+1. Install `podman` - on Fedora you can use:
+
+   ```
+   sudo dnf install podman
+   ``` 
+   
+### Testing that drenv is healthy
+
+Run this script to make sure `drenv` works. 
+
+   ```
+   cd ramen/test
+   scripts/drenv-selftest
+   ```
+   
+   Example output:
+   ```
+   1. Activating the ramen virtual environment ...
+
+
+   2. Creating a test cluster ...
+
+   2023-11-02 02:50:15,685 INFO    [test-f7c5e7f9-e976-4d18-9656-f70e7af46509test] Starting environment
+   2023-11-02 02:50:17,549 INFO    [test-f7c5e7f9-e976-4d18-9656-f70e7af46509cluster] Starting minikube cluster
+   2023-11-02 02:50:53,175 INFO    [test-f7c5e7f9-e976-4d18-9656-f70e7af46509cluster] Cluster started in 35.63 seconds
+   [...]
+   2023-11-02 02:51:11,359 INFO    [test-f7c5e7f9-e976-4d18-9656-f70e7af46509test] Environment started in 55.67 seconds
+
+   3. Deleting the test cluster ...
+
+   [...]
+
+   drenv is set up properly
+   ```
+
+## Starting the test environment
+
+Before using the `drenv` tool to start a test environment, you need to activate the python virtual environment:
+
+   ```
+   cd ramen
+   source venv
+   ```
+
+This environment simulates a Regional DR setup, when the managed clusters are located in different zones (e.g. US, Europe) and storage is replicated asynchronously. This environment includes 3 clusters - a hub cluster (`hub`), and 2 managed clusters (`dr1`, `dr2`) using `Ceph` storage. The connectivity between managed clusters will be configured using subctl (Submariner client). In addition, rbd-mirroring is configured using the Submariner connection for ceph image replication of block volumes.
```

Comment:
We use 72 characters per line limit for docs, see the devel-quick-start guide for reference. Not critical, we can improve this later.

Wrapping lines do not affect the output - the rendered html is the same. But the wrapped line are easier to work with for developers.

Reply:
Wrap in my editor has 80 characters with soft  wrap. Tried to reformat to 72 but really messed up document. Not changing to 72.